tests/test_plotting.py

```diff
@@ -410,6 +410,18 @@ def test_pie_plot_legend(plot_df):
     return fig
 
 
+@pytest.mark.mpl_image_compare(**MPL_KWARGS)
+def test_pie_plot_colors(plot_df):
+    fig, ax = plt.subplots(figsize=(8, 8))
+    with update_run_control(RC_TEST_DICT):
+        (
+            plot_df.filter(
+                variable="Primary Energy", model="test_model", year=2010
+            ).plot.pie(ax=ax, category="scenario", colors=["red", "blue"], title="foo")
```

Comment:
I know this is pedantic, but can you use values other than red and blue here so this test is easily visually distinguishable from the default values (e.g. in `test_pie_plot_labels.png`).

Reply:
Done